molecule/package_name/hosts.yml

```diff
@@ -0,0 +1,66 @@
+---
+cluster:
+  vars:
+    # common connection opts
+    ansible_user: root
+    ansible_connection: docker
+    become: true
+    become_user: root
+
+    # common cartridge opts
+    cartridge_app_name: some-app-name
+    cartridge_cluster_cookie: secret-cookie
+
+    cartridge_package_name: myapp
```

Comment:
For what `cartridge_app_name` variable?
For what `cartridge_package_name` variable?
Can we select `cartridge_package_name` from TGZ archive?